server/server.js

```diff
@@ -1,35 +1,35 @@
 import express from 'express'
-import dotenv from 'dotenv';
+import dotenv from 'dotenv'
 import cors from 'cors'
-import cookieParser from 'cookie-parser';
+import cookieParser from 'cookie-parser'
 
-import {connect} from './src/db/connetDB.js'
-import {authRouter} from './src/routes/auth.routes.js'
+import { connect } from './src/db/connetDB.js'
+import { authRouter } from './src/routes/auth.routes.js'
 import { homePageLogin } from './src/controllers/auth.controller.js'
 import { getMovies } from './src/controllers/movies.controller.js'
-import {movieRouter} from './src/routes/movie.routes.js'
+import { movieRouter } from './src/routes/movie.routes.js'
 
-const app=express();
-dotenv.config();
+const app = express()
+dotenv.config()
 
-const PORT=process.env.PORT || 5000;
+const PORT = process.env.PORT || 5000
```

Comment:
variable names in js use camel case, meaning you should name this to "port"
so `const port = proces.env.PORT...`